### DISCUSSION TITLE
arm64 docker configuration

### DISCUSSION THREAD

**Discussion by nickmcgill** [bluenviron/mediamtx#733] [Questions] [answered]
Hi,
I am trying to use the docker image on a Pi4, and I can get the container to run, but I cannot get at the rtsp-simple-server.yml file.
Using the line from the docs (yes, I did read them)
`docker run -d --network=host -v $PWD/rtsp-simple-server.yml:/rtsp-simple-server.yml aler9/rtsp-simple-server` creates a directory called rtsp-simple-server.yml in my home directory. The docker docs state that the -v flag is for mapping explicit paths (not files).
I cannot map the root directory in the container to an external directory.

Can anyone suggest how I should proceed (or if I'm just stupid)? 

Thanks in advance.

**Comment by aler9** [comment, marked as answer]
Hello, first create the configuration file in your home directory, then call the command.

**Reply by nickmcgill** [reply]
That was simple! Thank you.

[a comment by an automated account is not shown]
